server/src/main/java/io/crate/expression/operator/AllOperator.java

```diff
@@ -76,6 +76,15 @@ public static void register(Functions.Builder builder) {
                     )
             );
         }
+        builder.add(
+            AllEqOperator.SIGNATURE,
+            (signature, boundSignature) ->
+                new AllEqOperator(
+                    signature,
+                    boundSignature,
+                    result -> result == 0
+                )
```

Comment:
```suggestion
            AllEqOperator::new
```

See other suggestion further below